### ISSUE TITLE
Чистая консоль. Warnings

### ISSUE THREAD

**Issue by viktorovich87** [asd95/tt-adminui#11]
После сдачи проекта в консоле остались варнинги..
warnings - в файле Chart.js - service должен быть в колбеке у useEffect и находится тут
useEffect(() => {...},[service])  тогда варнингов не будет

**Comment by asd95** [comment]
Да, он там должен быть))) Я не указал его явно. Когда делал, много начитался про useEffect. В итоге выявил что указывать service в качестве подписки useEffect  не нужно. Он запускается только один раз для иммитирования POST каждые 25 секунд. Все что нужно, он должен очистить этот интервал когда компонент не отображается на странице. 
Или я что-то упустил. Подправьте пожалуйста. Спасибо
